htdocs/compta/bank/class/account.class.php

```diff
@@ -650,6 +650,14 @@ public function create(User $user, $notrigger = 0)
 			return -1;
 		}
 
+		$balance = $this->balance;
+		if (empty($balance) && !empty($this->solde)) {
+			$balance = $this->solde;
+		}
+		if (empty($balance)) {
+			$balance = 0;
```

Comment:
`$balance` is declared as float.
if we use `declare(strict_types=1);` in the future this code can raise an error
so you should use `$this->balance = 0.0;` instead

Reply:
@hansemschnokeloch are you sure ?

```
echo (float)0.00 . "\n";
echo (float)0 . "\n";
```

Reply:
My bad. I thought that we were in the opposite situation float assigned to int
```
declare(strict_types=1);
error_reporting(E_ALL);
class myClass
{
    public int $balance;
}

$c = new myClass();
$c->balance = 0.0; // Fatal error: Uncaught TypeError: Cannot assign float to property myClass::$balance of type int
```